### DISCUSSION TITLE
When will the next release be?

### DISCUSSION THREAD

**Discussion by robp94** [smallrye/smallrye-graphql#1920] [Q&A] [answered]
@jmartisk can you give some time range for 2.5?

**Comment by jmartisk** [comment, marked as answer]
I aim to get this into Quarkus 3.5. I usually release SmallRye about a week before the CR1 of the target Quarkus release to leave some time for the upgrade. Please follow this document https://github.com/quarkusio/quarkus/wiki/Release-Planning for the release schedule of Quarkus. 3.5 isn't listed yet, but I assume sometime second half of October...

**Reply by robp94** [reply]
Sounds good, thx.
